app/frontend/src/features/messages/groupchats/GroupChatsTab.tsx

```diff
@@ -1,49 +1,70 @@
-import { Box } from "@material-ui/core";
+import { Box, List } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import React, { useState } from "react";
+import React from "react";
 import { useQuery } from "react-query";
 import Alert from "../../../components/Alert";
 import CircularProgress from "../../../components/CircularProgress";
 import { Error as GrpcError } from "grpc-web";
 import { GroupChat } from "../../../pb/conversations_pb";
 import CreateGroupChat from "./CreateGroupChat";
-import GroupChatList from "./GroupChatList";
-import GroupChatView from "./GroupChatView";
 import { service } from "../../../service";
+import { messagesRoute } from "../../../AppRoutes";
+import TextBody from "../../../components/TextBody";
+import GroupChatListItem from "./GroupChatListItem";
+import { Link } from "react-router-dom";
 
-const useStyles = makeStyles({ root: {} });
+const useStyles = makeStyles((theme) => ({
+  root: {},
+  list: {
+    //margin won't go on the right, so make the width longer
+    width: `calc(100% + ${theme.spacing(4)})`,
```

Comment:
It looks fine for me without this set and it's creating an unnecessary horizontal scroll bar on tablet size width. What was this trying to fix at first?

Reply:
It's so the darkening background extends out to the side a bit. I'll hide the overflow.

Reply:
Is this because of the fixed width container globally on the page? Wonder if we should review whether that's still a good idea at a later stage if that's the case, since I remember Aapeli mentioning wanting to maybe do a full window/page width map before.